src/main/java/uk/gov/ons/ctp/response/action/endpoint/ActionRuleEndpoint.java

```diff
@@ -119,6 +121,38 @@
     return ResponseEntity.created(URI.create(newResourceUrl)).body(actionRuleDTO);
   }
 
+  /**
+   * PUT to update the specified Action Rule.
+   *
+   * @param actionRuleId            Id of the Action Rule to update
+   * @param actionRulePutRequestDTO Incoming ActionRuleDTO with details to update
+   * @param bindingResult       collects errors thrown by update
+   * @return ActionRuleDTO Returns the updated Action Rule details
+   * @throws CTPException            if update operation fails
+   * @throws InvalidRequestException if binding errors
+   */
+  @RequestMapping(value = "/{actionRuleId}", method = RequestMethod.PUT, consumes = "application/json")
+  public ActionRuleDTO updateActionRule(@PathVariable("actionRuleId") final UUID actionRuleId,
+                                @RequestBody(required = false) @Valid final ActionRulePutRequestDTO actionRulePutRequestDTO,
+                                final BindingResult bindingResult) throws CTPException, InvalidRequestException {
+    log.info("Updating Action Rule with {} - {}", actionRuleId, actionRulePutRequestDTO);
+    if (bindingResult.hasErrors()) {
+      throw new InvalidRequestException("Binding errors for update action rule: ", bindingResult);
+    }
+
+    ActionRule actionRuleToUpdate = mapperFacade.map(actionRulePutRequestDTO, ActionRule.class);
+    actionRuleToUpdate.setId(actionRuleId);
+    actionRuleToUpdate = actionRuleService.updateActionRule(actionRuleToUpdate);
```

Comment:
:art: I would generally suggest not assigning the product of a method to a variable that's used within that method, mainly for readability. I found this code much harder to understand at a quick browse until I realised you were reassigning aRTU with the product of updateActionRule